README.md

```diff
@@ -70,3 +70,16 @@ Or only with the workload:
 ```ShellSession
 $ dci-pipeline-check https://github.com/dci-labs/<your company>-<lab>-config/pull/1 $KUBECONFIG workload
 ```
+
+# Certification with DCI
+
+In the pipelines/certification-pipeline.yml file, you will find a stub configuration to certify containers, Helm charts, operators, and create CNF projects. Customize it by providing your credentials and certification items, and then run it using the following command:
```

Comment:
it would be cool to have the `pipelines/certification-pipeline.yml` file as a link pointing to the template you are adding.

Reply:
thank you @manurodriguez added the link